### ISSUE TITLE
What will happen if flash storage quota is exceeded while debug is false?

### ISSUE THREAD

**Issue by Seedmanc** [nfriedly/Javascript-Flash-Cookies#30]
Normally, I assume, a dialog window would appear over the flash plugin area that requested more free space that it was initially allowed. But with debug=false being default, flash window is out of sight. How will a user be notified that there is a need to change quota?

**Comment by nfriedly** [comment]
Hum, that's a good point. Does flash have an API to determine how close you are to the limit, or is it just the popup?

**Comment by Seedmanc** [comment]
I don't really know much about flash coding, and a quick search gave me nothing about requesting current size limit.
However, from what I see in your Storage.as and in console output with debug=true, whenever the limit is reached, a message is sent to console and you can detect user actions (whether the quota was set enough to write or denied). 
Now, the thing is that the default debug value is false and if it was changed to true, then the potential user is most likely tech-savvy enough already to know what's up and to take a look at the console if the limit was encountered. For majority of users though the debug will be false and they would not even know that the data is not being saved due to quota limit. 
Perhaps you should try to notify user somehow when the flushStatus is "Pending" but at the same time debug is false. Maybe a js alert with a link to Flash settings page about LSO size or maybe try and bring the flash window into view, I dunno.

**Comment by nfriedly** [comment]
Right. I might try to add something like that at some point... but for right now I'm just going to add a note to the readme.
